web/packages/core/src/ruffle-player.ts

```diff
@@ -463,6 +468,15 @@ export class RufflePlayer extends HTMLElement {
                     "'{secs: 15, nanos: 0}'."
             );
         }
+        if (
+            this.loadedConfig &&
+            (this.loadedConfig.contextMenu === false ||
```

Comment:
nit: This can be simplified to `typeof this.loadedConfig.contextMenu === "boolean"`.